nextjs-13/src/stylesheets/components/_list-newsletter.scss

```diff
@@ -0,0 +1,23 @@
+.list-newsletter {
+  font-weight: var(--font-weight-5);
+  width: 500px;
```

Comment:
Always prefer to use `rem` for unit sizing

Reply:
Resolved [here](https://github.com/nimblehq/js-framework-benchmark/pull/110/commits/fe769187ea84c37f4b04d3ba993450abd5940431#diff-099e542ab2be0b36ba6ee7666ef7e5ef285c13f7a6e3e70bb3c854526d41b0e1R3)